### PR TITLE
Tripur.html

### PR DESCRIPTION
**Pull request by tripur3121**

### CODE REVIEW

File: Tripur.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                <a href="#projects" class="nav-link">Projects</a>
+                <a href="#skills" class="nav-link">Skills</a>
+                <a href="#contact-us" class="nav-link">Contact Us</a>
+            </div>
+        </nav>
+
+        <!-- HomePage Section -->
+
+        <div class="home-container" id="home">
+            <h1 class="name">saksham somani</h1>
```
Comment (RITEKROUNAK) [inline Tripur.html:37]: Nice Try.

File: Tripur.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                    say hi, please feel free to email me at,
+                    <span
+                        ><a href="mailto:miten3377@gmail.com"
+                            >elonmusk@gmail.com</a
+                        ></span
+                    >
+                </p>
+            </div>
+            <div class="credits">
+                <strong> Made with 💖 by Tripur S Joshi. </strong>
```
Comment (RITEKROUNAK) [inline Tripur.html:214]: Perfect 👍

**Review by RITEKROUNAK — CHANGES_REQUESTED** [review]
Atleast Edit Your Name.

File: Tripur.html [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        <title>Portfolio</title>
+    </head>
+
+    <body>
+        <!-- Navbar section -->
+
+        <nav class="fixed-top navbar">
+            <!-- Navbar brand -->
+            <a href="#home" class="nav-link brand"
+                ><strong> &lt;saksham /&gt; </strong></a
```
[the remark this answers is not in the document]
Comment (tripur3121) [reply Tripur.html]: Done.

**Review by RITEKROUNAK — APPROVED** [review]

**Pull request merged**